lib/ContentTypeParser.js

```diff
@@ -65,7 +77,8 @@ ContentTypeParser.prototype.getParser = function (contentType) {
 ContentTypeParser.prototype.run = function (contentType, handler, request, reply) {
   var parser = this.cache.get(contentType) || this.getParser(contentType)
   if (parser === undefined) {
-    reply.code(415).send(new Error('Unsupported Media Type: ' + contentType))
+    var err = new FST_ERR_CTP_INVALID_MEDIA_TYPE(`Unsupported Media Type: ${contentType}`)
+    reply.code(415).send(err)
```

Comment:
Should we include the code in the Error instead?

Reply:
Example?

Reply:
`err.core = 415`

Reply:
You mean `err.statusCode = 415` ?

Reply:
yeah